arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dtsi

```diff
@@ -155,18 +155,18 @@
 		status = "disabled";
 	};
 
-	supervisor: supervisor@10 {
-		compatible = "technologic,supervisor";
+	wizard: wizard@10 {
+		compatible = "technologic,tswizard";
```

Comment:
I see other mentions of `wizard-adc` `wizard-temp` etc., this is the first use of `tswizard`. Since we didn't use `tssupervisor`, i.e. the change is from `supervisor`, the change should be to `wizard` for consistency. Either that, or other instances of `wizard` like `-adc` should probably move to `tswizard-adc` and `tswizard-temp`. Though I prefer dropping the `ts` on there since the full string already includes the manufacturer, e.g. `technologic,wizard`

Reply:
The original strategy was to exchange like-for-like, but it turned out the original code uses nearly all permutations of ts and supervisor with and without caps, hyphens, and underscores.  So the strategy was tswizard would be used to define our namespace, and wizard would be used where the namespace is already defined.